README.md

```diff
@@ -71,6 +71,14 @@ The default Scala version supported is 2.11. To build for Scala 2.12 version, bu
 mvn clean package -DskipTests -DskipITs -Dscala-2.12
 ```
 
+### Build without spark-avro module
+
+The default hudi-jar bundles spark-avro module. To build without spark-avro module, build using `spark-shade-unbundle-avro` profile
```

Comment:
can we also add the `--package`  instruction here to explicitly say how to add spark-avro if building without it.